facebook/react-native
# [iOS][Bug] - fix crash on performance logger

## Summary

Fix Issue https://github.com/facebook/react-native/issues/24820

It's caused by `_labelsForTags` and `RCTPLTag` being out of sync, the crash might only be one of the issues that this bug was causing.

## Changelog

[iOS] [Fixed] - fix crash on performance logger

## Test Plan

- Launch iOS simulator
- Enable performance monitor
- Tap on performance monitor

It should not crash
